### ISSUE TITLE
Custom scroll behavior

### ISSUE THREAD

**Issue by cwooldridge1** [amcharts/amcharts5#405]
Is there a way to create custom scroll behavior? I noticed on some other chart providers such as high chart and trading view there's this scrolling behavior that keeps the right side in the same x-axis position (x-axis value the furthest right is constant) but the left side x-axis position (x-axis value furthest to the left) increases and decreases on scroll. The right position stays locked on scroll and only changes if the user drags with their mouse. This is by far my favorite scroll behavior for time series charts and I'm wondering if there is a way to emulate it. I realize that in words you may not be able to fully understand what I am depicting so here is a link to a trading view chart that has the same scroll behavior as I am explaining: https://www.tradingview.com/chart

**Comment by zeroin** [comment]
Do you mean mouse wheel behavior? If yes, then I would say our approach is better, as it zooms exactly to the place where the mouse is located. So if you move the mouse to the very right of the chart and rotate the wheel, it will behave like in tradingview demo. And I don't see a way to change it.

**Comment by cwooldridge1** [comment]
Yeah, a lot of financial-based platforms have been adopting that mouse wheel behavior and making it a norm in the realm of financial charting. Because of this feature's popularity in the finance industry, it may be something to think about/backlog if you guys decided to put more time into the financial charts but regardless thanks for your time.

**Comment by martynasma** [comment]
Re-opening and refactoring to an enhancement, as we might consider implementing it at some point.

**Comment by martynasma** [comment]
Just noticing this hasn't been closed.

It has been possible using `XYChart` (or `StockPanel`) setting `wheelZoomPositionX`.

Docs: https://www.amcharts.com/docs/v5/charts/xy-chart/zoom-and-pan/#Wheel_zoom_center
